wbooks/src/app/components/bookComment/styles.ts

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+import { white, tundora, dustyGray } from '@constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    backgroundColor: white,
+    flex: 1,
+    flexDirection: 'row',
+    padding: 20
+  },
+  image: {
+    borderRadius: 50,
+    height: 50,
+    width: 50
```

Comment:
Extrae el 50 a una constante y lo usas.

Reply:
the variable was extracted to a constant